stl/inc/yvals_core.h

```diff
@@ -1124,11 +1124,7 @@
 #define __cpp_lib_bind_front        201907L
 #define __cpp_lib_bit_cast          201806L
 

```

Comment:
Because the conditional compilation is being removed, this empty line should be removed (which will cause clang-format to realign whitespace).